conf/tai.routes

```diff
@@ -137,28 +133,24 @@ POST        /end-company-car/check-your-answers
-GET         /taxable-income                                                            controllers.TaxFreeAmountControllerNew.taxFreeAmount
-GET         /your-tax-free-amount                                                      controllers.TaxFreeAmountControllerNew.taxFreeAmount
+GET         /tax-free-allowance                                                        controllers.TaxFreeAmountController.taxFreeAmount
+GET         /taxable-income                                                            controllers.TaxFreeAmountController.taxFreeAmount
+GET         /your-tax-free-amount                                                      controllers.TaxFreeAmountController.taxFreeAmount
 
 GET         /tax-codes                                                                 controllers.YourTaxCodeController.taxCodes
-GET         /paye-income-tax-estimate                                                  controllers.EstimatedIncomeTaxControllerNew.estimatedIncomeTax
+GET         /paye-income-tax-estimate                                                  controllers.EstimatedIncomeTaxController.estimatedIncomeTax
 
 
 GET         /income-tax-comparison                                                     controllers.IncomeTaxComparisonController.onPageLoad
 
-GET         /your-income-calculation                                                   controllers.YourIncomeCalculationController.yourIncomeCalculationPage(id: Option[Int])
-GET         /print-your-income-calculation                                             controllers.YourIncomeCalculationController.printYourIncomeCalculationPage(id: Option[Int])
 
-GET         /your-income-calculation-details/:id                                       controllers.YourIncomeCalculationControllerNew.yourIncomeCalculationPage(id: Int)
-GET         /print-your-income-calculation-details/:id                                 controllers.YourIncomeCalculationControllerNew.printYourIncomeCalculationPage(id: Int)
+GET         /your-income-calculation-details/:id                                       controllers.YourIncomeCalculationController.yourIncomeCalculationPage(id: Int)
+GET         /print-your-income-calculation-details/:id                                 controllers.YourIncomeCalculationController.printYourIncomeCalculationPage(id: Int)
 
-GET         /your-income-calculation-previous-year/:empId                              controllers.YourIncomeCalculationController.yourIncomeCalculationPreviousYearPage(empId: Int)
-GET         /print-your-income-calculation-previous-year/:empId                        controllers.YourIncomeCalculationController.printYourIncomeCalculationPreviousYearPage(empId: Int)
+GET         /your-income-calculation-historic-years/:year/:empId                       controllers.YourIncomeCalculationController.yourIncomeCalculationHistoricYears(year: uk.gov.hmrc.tai.model.tai.TaxYear, empId: Int)
```

Comment:
Can you either revert this url back or make sure PTA doesn't break as htey are calling this url